x-pack/plugins/saved_objects_tagging/public/management/components/table.tsx

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useRef, useEffect, FC } from 'react';
+import { EuiInMemoryTable, EuiLink } from '@elastic/eui';
+import { Action as EuiTableAction } from '@elastic/eui/src/components/basic_table/action_types';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { TagsCapabilities, TagWithRelations } from '../../../common';
+import { TagBadge } from '../../components';
+
+interface TagTableProps {
+  loading: boolean;
+  capabilities: TagsCapabilities;
+  tags: TagWithRelations[];
+  selectedTags: TagWithRelations[];
+  onSelectionChange: (selection: TagWithRelations[]) => void;
+  onEdit: (tag: TagWithRelations) => void;
+  onDelete: (tag: TagWithRelations) => void;
+  getTagRelationUrl: (tag: TagWithRelations) => string;
+  onShowRelations: (tag: TagWithRelations) => void;
+}
+
+const tablePagination = {
+  initialPageSize: 20,
+  pageSizeOptions: [5, 10, 20, 50],
+};
+
+const sorting = {
+  sort: {
+    field: 'name',
+    direction: 'asc' as const,
+  },
+};
+
+export const isModifiedOrPrevented = (event: React.MouseEvent) =>
+  event.metaKey || event.altKey || event.ctrlKey || event.shiftKey || event.defaultPrevented;
+
+export const TagTable: FC<TagTableProps> = ({
+  loading,
+  capabilities,
+  tags,
+  onSelectionChange,
+  selectedTags,
+  onEdit,
+  onDelete,
+  onShowRelations,
+  getTagRelationUrl,
+}) => {
+  const tableRef = useRef<EuiInMemoryTable<any>>();
+
+  useEffect(() => {
+    if (tableRef.current) {
+      tableRef.current.setSelection(selectedTags);
+    }
+  }, [selectedTags]);
+
+  const actions: Array<EuiTableAction<TagWithRelations>> = [];
+  if (capabilities.edit) {
+    actions.push({
```

Comment:
Can we add an `'aria-label'` field to these actions with their unique tag names? Something like 
`Edit {tagName} tag`.

Reply:
EUI already kinda handle that with an `aria-labelledby` pointing to an element with the `name` of the action.

<img width="1076" alt="Screenshot 2020-10-20 at 18 52 43" src="https://user-images.githubusercontent.com/1532934/96618624-78f4b780-1305-11eb-9afb-1d49bb95f065.png">
<img width="647" alt="Screenshot 2020-10-20 at 18 52 49" src="https://user-images.githubusercontent.com/1532934/96618630-7a25e480-1305-11eb-887f-e38678aca01c.png">

Also the action (`EuiTableAction`) does not support the custom `aria-label` I believe? 

<img width="580" alt="Screenshot 2020-10-20 at 18 55 05" src="https://user-images.githubusercontent.com/1532934/96618952-dc7ee500-1305-11eb-8eef-5d479cf96d02.png">

Should I change the name from `Edit` to `Edit {tagName} tag` and do the same for the `delete` action?